docs/user-guide/sdh/release-hub.md

```diff
@@ -0,0 +1,238 @@
+
+7. Enter a [semantic version](https://semver.org/) in **Release Version** field, e.g., `1.0.0`
+
+    ![Figure 5: Versioning the Release](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/semantic-versioning.jpg)
+
+8. (Optional) Give a name to the release, e.g., `numero-beta`. If you don’t provide one, the name will be same as release version (i.e., 1.0.0).
+
+9. (Optional) Add a description of the release.
+
+10. Click **Create Release**.
+
+    ![Figure 6: Saving Release Details](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/release-version.jpg)
+
+If you are creating your first release, you may proceed with the **Create from scratch** option. However, for subsequent versions of your release (say 1.0.1), you may clone an existing release (e.g., 1.0.0) as shown below. Please note, you can only clone releases belonging to the same track.
+
+![Figure 7: Cloning an Existing Release](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/clone-release.jpg)
+
+---
+
+## Adding Applications
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add applications to a release track.
+{% endhint %}
+
+This involves the inclusion of applications you wish to rollout in the release version created by you.
+
+1. Click **+ Add Application** button present within the release you created.
+
+    ![Figure 8: Adding Apps to Release](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-apps.jpg)
+
+2. Click the **Search and add applications** dropdown.
+
+3. Use the checkbox to add applications from your projects.
+
+    ![Figure 9: Choosing Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/search-apps.gif)
+
+4. Click **Add Release Stage**.
+
+    ![Figure 10: Dividing Release in Stages](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-release-stage.jpg)
+
+{% hint style="info" %}
+### Importance of Release Stages
+By default, your selected applications will be set to release in one go. However, you can also release them in stages. In other words, you can decide which set of applications should be released first, subsequently which ones to release next, and the ones to release last. <br /><br /> For example, if you have a new login feature (backend) and a dashboard update (frontend), you might want to release the login feature first to ensure users can log in before seeing the updated dashboard. 
+{% endhint %}
+
+5. Use the drag-and-drop feature to move applications from one stage to another.
+
+    ![Figure 11: Rearranging the Sequence](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/release-stage-v2.gif)
+
+{% hint style="warning" %}
+The drag-and-drop feature is not meant for changing the sequence of applications within a stage. Use it solely for moving applications from one release stage to another.
```

Comment:
This feels like, it's not meant to but it can? Should we say like it won't change in same stage but will across?

Reply:
No, @abhibhaw 
I rephrased it now, but the original intention was exactly that!
The drag-and-drop is not meant to but it can change the app sequence in same stage...So writing 'won't change in same stage' would be incorrect. I and @uxarya-d have discussed this earlier.

Reply:
This is rephrased line I am proposing "The drag-and-drop feature is designed specifically for moving applications between different release stages. It is not meant to alter the sequence of applications within the same stage."